bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -911,6 +913,37 @@ impl Room {
         room_event_cache.clear().await?;
         Ok(())
     }
+
+    /// Update the canonical alias of the room.
+    ///
+    /// Note that publishing the alias in the room directory is done separately.
+    pub async fn update_canonical_alias(
+        &self,
+        new_alias: Option<String>,
+    ) -> Result<(), ClientError> {
+        let new_alias = new_alias.map(TryInto::try_into).transpose()?;
+        self.inner.update_canonical_alias(new_alias).await.map_err(Into::into)
+    }
+
+    /// Enable End-to-end encryption in this room.
+    pub async fn enable_encryption(&self) -> Result<(), ClientError> {
+        self.inner.enable_encryption().await.map_err(Into::into)
+    }
+
+    /// Update room history visibility for this room.
+    pub async fn update_history_visibility(
```

Comment:
I see we have a way to update this API in FFI now nice, but is there a way to just read the history visibility in FFI? (like from the `Room` or the `RoomInfo`)?

Reply:
I just added it on https://github.com/matrix-org/matrix-rust-sdk/pull/4401/commits/17530d2caf5c3d3fa03dcc543c90755ce29923fb